Sources/Shared/API/ServerManager.swift

```diff
@@ -11,6 +11,7 @@ public protocol ServerObserver: AnyObject {
 public protocol ServerManager {
     var all: [Server] { get }
     func server(for identifier: Identifier<Server>) -> Server?
+    func server(for identifier: Identifier<Server>, fallback: Bool) -> Server?
```

Comment:
since this one is going to behave slightly different than the other fallbacks, can we rename it so it implies more of a 'guaranteed'-ish return?